directory/src/main/java/org/entcore/directory/controllers/UserController.java

```diff
@@ -170,6 +171,9 @@ public void handle(UserInfos user) {
 											if (userBeforeUpdate != null && userBeforeUpdate.result() != null) {
 												sendEmailOrMobileUpdateNotifications(request, userBeforeUpdate.result(), body);
 											}
+											if (!userPositionIds.isEmpty()) {
```

Comment:
```suggestion
											if (userPositionIds!=null) {
```
Attention à bien distinguer les 3 cas de figures : 
- aucun query param `positionIds` => le graphe ne doit pas être modifié, le User conserve ses liens existant avec tout noeud UserPosition.
- `positionIds` sans valeur => équivaut à un tableau vide, il faut supprimer tout lien existant entre le User et un noeud UserPosition.
- `positionIds` avec une ou plusieurs valeurs => ajouter les liens manquants, et supprimer ceux existant qui ne sont pas dans le tableau.

En pratique, les 2 derniers cas sont bien gérés par le code proposé. **Il manque à gérer le 1er cas.**

Reply:
Done